cdb2api/cdb2api.c

```diff
@@ -4763,7 +4763,7 @@ int cdb2_open(cdb2_hndl_tp **handle, const char *dbname, const char *type,
         char *p;
         p = strchr(type, ':');
         if (p) {
-            strcpy(hndl->hosts[0], type);
+            strncpy(hndl->hosts[0], type, sizeof(hndl->hosts[0]));
```

Comment:
@akshatsikarwar I'll take a look at bb/str0.c, I was actually looking for `misc` string functions.
@riverszhang89, @adizaimi  you are right, the -1 will suffice.